client/verta/verta/_dataset_versioning/dataset.py

```diff
@@ -121,3 +143,9 @@ def create_path_version(self):  # TODO: same params as Path.__init__()
 
     def get_latest_version(self):
         raise NotImplementedError
+
+    def _update(self, msg, response_proto, endpoint):
+        self._fetch_with_no_cache()
```

Comment:
I don't think we need this, since`Dataset` doesn't use the get-entire-msg-and-send-it-back mechanism that Registry entities do. So this would be an unnecessary backend call per write.
```suggestion
```

Reply:
Yeah you're right. Fixed.